### ISSUE TITLE
False positive for `Symbol.asyncIterator()` in iOS Safari

### ISSUE THREAD

**Issue by MetRonnie** [amilajack/eslint-plugin-compat#562]
According to https://caniuse.com/mdn-javascript_builtins_symbol_asynciterator it is available since iOS 11.3. However even if I set `not iOS < 16` in `.browserslistrc` I still get
```
error  Symbol.asyncIterator() is not supported in iOS Safari 16.0  compat/compat
```

Also, it seems the plugin prints the _lowest_ version of the browser that lacks compatibility rather than the highest. The highest version lacking support is of more concern.

**Comment by koddsson** [comment]
Hey! I just tried to reproduce this [here](https://github.com/koddsson/eslint-plugin-compat-demo/tree/issue-562) but I'm unable to get the same error. Maybe you can make a PR to that branch showcasing the issue?

Note that I'm using the latest version (4.1.4) which might have fixed this.

**Comment by MetRonnie** [comment]
I can no longer reproduce
